src/backend/marsha/core/tests/test_views_lti_document.py

```diff
@@ -59,7 +59,8 @@ def test_document_view_instructor(self, mock_get_consumer_site, mock_verify):
 
         # Extract the file data
         data_file = re.search(
-            '<div class="marsha-frontend-data" id="document" data-document="(.*)">',
+            '<div class="marsha-frontend-data" id="document" '
```

Comment:
I understand the logic but this seems very redundant to me. How did we manage this before for videos?

Don't you think we should standardize this all across instead of running with the two names in parallel? @sampaccoud do you have an opinion on this?

Reply:
you mean `id` and `data-modelname` ?

Reply:
Yes, those two

Reply:
The problem are the model types. Some are pluralize (videos and timed text tracks), orther are not (thumbnail). IMOP a model name is never pluralize, we should change that and keep the `id`.

Reply:
Please check this commit https://github.com/openfun/marsha/pull/487/commits/7b0889b7f9e5a76fa3c1abe52490ad4eb6e4ef01 What do you think ?

Reply:
Looks good to me 👍